source/system/structs/VectorList.ooc

```diff
@@ -51,9 +51,11 @@ VectorList: class <T> extends List<T>{
 		this _count -= 1
 		result
 	}
-	removeAt: override func (index: Int) {
-		this _vector copy(index + 1, index)
-		this _count -= 1
+	removeAt: override func (index: Int) { this removeRange(index, index + 1) }
```

Comment:
No surrounding function is a one-liner, so this should according to the style guide not be a one-liner either.